packages/synapse-react-client/src/types/markdown-it-plugins.d.ts

```diff
@@ -1,10 +1,4 @@
-declare module '*.svg' {
```

Comment:
Split the type definitions / augmentations in `types.d.ts` into multiple files based on the purpose of the types